### ISSUE TITLE
Looks like there isn't a way to configure the whitelist for the gleak.middleware instance created internally?

### ISSUE THREAD

**Issue by shzmilydahlke** [aheckmann/gleak#8]
I want to use the express api middleware to have gleak run on every http request, but doesn't look like this use is designed to allow whitelisting (as the Gleak instance is internal to the method and inaccessible).
Is there a way to open up this path to be whitelist configurable?

**Comment by aheckmann** [comment]
yeah sounds good. we could refactor the `middleware` method to accept an options object and add a new property for the whitelist option or similar. PR?
